pkg/cgroup/cgroup.go

```diff
@@ -389,10 +390,12 @@ func GetCgroupControllerHierarchy(subsys string) (int, error) {
 // given cgroupId and subPath (related to cgroup fs root dir). If subPath is
 // empty, then all directories from cgroup fs will be searched for the given
 // cgroupId.
-func GetCgroupPath(rootDir string, cgroupId uint64, subPath string) (string, error) {
+//
+// Returns found cgroup path, its ctime, and an error if relevant
```

Comment:
nit: godoc formatting requires function name as the first string (I know this is a backport and has been merged already).